src/utils/testfiles/mockCristinPersonSearch.ts

```diff
@@ -60,6 +60,7 @@ export const mockCristinPersonSearch: SearchResponse<CristinPerson> = {
         en: 'Background',
       },
       keywords: [],
+      nvi: { verifiedAt: { id: '' }, verifiedBy: { id: '' } },
```

Comment:
dette er en kopi av `emptyNviVerification`. Kan gjenbruke den i stedet, så slipper vi vedlikehold begge plasser
```suggestion
      nvi: emptyNviVerification,
```